.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Build and Optionally Release
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches:
+    - main
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v2
+      with:
+        image: tonistiigi/binfmt:latest
+        platforms: all
```

Comment:
So those are the defaults. In which case I'd suggest to remove those.

... However! We might want to specify the platforms that we use here. Not sure what impact that would have but might be a nice touch to only specify what we use.